gtda/images/filtrations.py

```diff
@@ -152,7 +152,7 @@ def transform(self, X, y=None):
 
         """
         check_is_fitted(self)
-        Xt = check_array(X,  ensure_2d=False, allow_nd=True, copy=True)
+        Xt = check_array(X, ensure_2d=False, allow_nd=True, copy=True)
```

Comment:
So these can work with 1D input then?

Reply:
Thanks @rth! Indeed you are right to wonder: they should not IMO. I have changed everything in that module and check remaining instances throughout the library. I believe the remaining usages in [`time_series/preprocessing.py`](https://github.com/giotto-ai/giotto-tda/blob/master/gtda/time_series/preprocessing.py), [`time_series/embedding.py`](https://github.com/giotto-ai/giotto-tda/blob/master/gtda/time_series/embedding.py), [`mapper/filter.py`](https://github.com/giotto-ai/giotto-tda/blob/master/gtda/mapper/filter.py) and [`mapper/cover.py`](https://github.com/giotto-ai/giotto-tda/blob/master/gtda/mapper/cover.py) are good.